androidApp/src/main/java/com/EENX15_22_17/digital_journal/android/ui/suicideassessment/service/DrawCheckBoxes.kt

```diff
@@ -0,0 +1,50 @@
+package com.EENX15_22_17.digital_journal.android.ui.components_beta
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.Text
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.EENX15_22_17.digital_journal.android.ui.suicideassessment.service.SummaryService
+import com.EENX15_22_17.digital_journal.android.ui.theme.colorTextBlackLight
+
+// Suicideassessment "ViewModel"
+@Composable
+fun DrawCheckBoxes(items: List<CheckBoxData>, whichSection : String ) {
```

Comment:
Even though composables are functions, they are not used as such. Name them according to what they represent instead of what they do, such as `CheckboxSection` or something similar.